### PR TITLE
Added the 3rd option for showing per country covid data

### PR DESCRIPTION
**Pull request by stephin007**
> 🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

### Please check if the PR fulfills these requirements

- [x] Make sure you are requesting to **NEW-UI**. Don't request other protected Branches like **staging/master**
- [x] Make sure no conflicts are present in the code, if so please resolve it(_Tip: Always fetch upstream_)
- [x] Your Commit messages should make sense.
- [x] Don't push your package.lock.json as this project uses yarn.lock already.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Describe your changes

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Ennhancement

- **What is the current behavior?** (You can also link to an open issue here)
Does not show country covid data.

- **What is the new behavior (if this is a feature change)?**
Added extra option in the CovidWorldData for showing country and further for each country data will be shown.

- **Does this PR introduce a breaking change?** (i.e changes that may require users to update/refactor existing instances of the application)
NO

- **Other Information**:
NONE

❤️ Thank you!

### CODE REVIEW

**Comment by stephin007** [comment]
@Justinnn07 can you please review this , but dont merge just approve it, if everything is looking okay to be merged

**Comment by Justinnn07** [comment]
> @Justinnn07 can you please review this , but dont merge just approve it, if everything is looking okay to be merged

Sure!

**Review by Justinnn07 — APPROVED** [review]
Nice work @stephin007 💥💥

**Pull request merged**
